File: rs/demo/src/code.rs
fn hello() {
    println!("hello world");
}

fn demo(bool parameter) {
    if parameter {
        println!("One");
    } else {
        println!("Two");
    }
}

fn factorial(i32 n) {
    let mut result = 1;
    for i in 1..n {
        result = result * i;
    }

    println!("{}", result);
}

File: py/code.py
def hello():
    print("hello world")

def demo(parameter):
    if parameter:
        print("one")
    else:
        print("two")

def factorial(n):
    result = 1
    for i in range(1, n + 1):
        result = result * i
    print(result)

hello()
demo(True)
demo(0)
demo(False)
factorial(10)